cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java

```diff
@@ -29,6 +29,12 @@
 
     public static synchronized void main(String[] args) throws Exception
     {
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
+        { 
+            System.out.println("This application needs at least Java 8 or 10.0.1!"); 
```

Comment:
```suggestion
            System.out.println("This application needs at least Java 8!"); 
```

This is enough, Java 10 is obsolete now.